packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerVisual.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { css } from '@emotion/react';
 import { from } from '@guardian/src-foundations/mq';
 import { ImageAttrs, ResponsiveImage } from '../../../shared/ResponsiveImage';
-import { Image } from '@sdc/shared/src/types';
```

Comment:
This is an unrelated change - but good to fix this and import from the right place!